test/common/router/config_impl_test.cc

```diff
@@ -2128,6 +2132,40 @@ TEST(RouteMatcherTest, TestDuplicateDomainConfig) {
                EnvoyException);
 }
 
+TEST(RouteMatcherTest, TestCaseSensitiveDomainConfig) {
+  std::string json = R"EOF(
```

Comment:
Bonus points for making this a YAML v2 test instead of v1 (see other examples).

Reply:
yup. Will move it to v2 !!!